modules/rabbitmq/manager.go

```diff
@@ -0,0 +1,222 @@
+package rabbitmq
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/CodeCollaborate/Server/utils"
+	"github.com/streadway/amqp"
+)
+
+/**
+ * RabbitMq manager for CodeCollaborate Server.
+ * @author: Austin Fahsl and Benedict Wong
```

Comment:
Try to put author tags in all of the files that you make, it's just good practice.